pkg/watch/watch.go

```diff
@@ -48,7 +53,9 @@ type WatchParameters struct {
 	// DevfileBuildCmd takes the build command through the command line and overwrites devfile build command
 	DevfileBuildCmd string
 	// DevfileRunCmd takes the run command through the command line and overwrites devfile run command
-	DevfileRunCmd string
+	DevfileRunCmd    string
```

Comment:
Thinking aloud here...

Could we make `regenerateAdapterAndPush` and `regenerateComponentAdapterFromWatchParams` methods on the `WatchOptions` type? Then I don't think we'd need to add `DevfileRunCmd`, `DevfilePath` and `DevfileNamespace` to the WatchParameters struct, no?

Reply:
Yep, that works, done.